controllers/bajas.go

```diff
@@ -89,61 +71,35 @@ func (c *BajaController) Post() {
 // @router /:id [put]
 func (c *BajaController) Put() {
 
-	defer func() {
-		if err := recover(); err != nil {
-			logs.Error(err)
-			localError := err.(map[string]interface{})
-			c.Data["mesaage"] = (beego.AppConfig.String("appname") + "/" + "BajaController" + "/" + (localError["funcion"]).(string))
-			c.Data["data"] = (localError["err"])
-			if status, ok := localError["status"]; ok {
-				c.Abort(status.(string))
-			} else {
-				c.Abort("500") // Error no manejado!
-			}
-		}
-	}()
+	defer errorctrl.ErrorControlController(c.Controller, "BajaController")
 
 	var id int
 	if v, err := c.GetInt(":id"); err != nil || v <= 0 {
 		if err == nil {
 			err = errors.New("Se debe especificar una baja válida")
 		}
-		logs.Error(err)
-		panic(map[string]interface{}{
-			"funcion": "Put - GetInt(\":id\")",
-			"err":     err,
-			"status":  "400",
-		})
+		panic(errorctrl.Error("Put", "c.GetInt(\":id\")", "400"))
 	} else {
 		id = v
 	}
 
 	var v *models.TrSoporteMovimiento
 	if err := json.Unmarshal(c.Ctx.Input.RequestBody, &v); err == nil {
+		panic(errorctrl.Error("Put", "json.Unmarshal(c.Ctx.Input.RequestBody, &v)", "400"))
```

Comment:
Faltaba negar la condición del `if`... ?
```suggestion
	if err := json.Unmarshal(c.Ctx.Input.RequestBody, &v); err != nil {
		panic(errorctrl.Error("Put - json.Unmarshal(c.Ctx.Input.RequestBody, &v)", err, "400"))
```